linera-base/src/identifiers.rs

```diff
@@ -960,6 +986,7 @@ impl Display for AccountOwner {
                 write!(f, "0x{}", hex::encode(&value.to_be_bytes()[..]))?
             }
             AccountOwner::Address32(value) => write!(f, "0x{}", value)?,
+            AccountOwner::Address20(value) => write!(f, "{}", value)?,
```

Comment:
`alloy_primitives::Address`' implements the `Display` that already prepends with `0x`.